plat/common/x86/acpi.c

```diff
@@ -256,3 +261,13 @@ acpi_madt_t *acpi_get_madt(void)
 
 	return acpi_madt;
 }
+
+/*
+ * Return the Fixed ACPI Description Table (FADT).
+ */
+acpi_fadt_t *acpi_get_fadt(void)
+{
+	UK_ASSERT(acpi_fadt);
```

Comment:
According to the documentation, it should be able to return `NULL`. But this assertion prevents this.

Reply:
Good point! Yes, I will admit I just copy-pasted this from the previously implemented `MADT` function :laughing: . In an ACPI system it is mandatory to have the `FADT`, so I think I will just remove the assertion, since it makes more sense for the caller to verify it (they already do), if assertions are enabled, because it must always be available.

This also seems to be the case for the `MADT`, now that I've mentioned it. I guess I will remove the assertion for both of them. Thanks!